src/gz21_ocean_momentum/data/pangeo_catalog.py

```diff
@@ -8,11 +8,6 @@
 
 from intake.config import conf
 
-# TODO variable currently unused? Remove? Or is it setting environment variable?
-conf["persist_path"] = "/scratch/ag7531/"
-# TODO variable currently unused. Remove?
-CACHE_FOLDER = "/scratch/ag7531/cm26_cache"
```

Comment:
We can probably remove everything related to CACHE_FOLDER. This was created so that if the data has already been downloaded locally in the cache folder, it uses that data. However, I had implemented this in a "dirty" way by modifying directly the intake library...